GovUkDesignSystem/GovUkDesignSystemComponents/CheckboxItem.cshtml

```diff
@@ -1,5 +1,3 @@
 ﻿@model GovUkDesignSystem.GovUkDesignSystemComponents.CheckboxItemViewModel
 
 @{ await Html.RenderPartialAsync("/GovUkDesignSystemComponents/Item.cshtml", Model); }
-
```

Comment:
Moving this to Checkboxes.cshtml so that there is only one hidden field, rather than one per checkbox (all checkboxes in a set have the same name)